src/providers/oauth-provider.js

```diff
@@ -11,6 +11,7 @@ var defaults = {
   clientId: null,
   clientSecret: null,
   grantPath: '/oauth2/token',
+  grantType: 'password',
```

Comment:
I would leave the grant_type of getRefreshToken hard coded and the default is still password. So it can't really break it :monkey_face: